libyul/backends/evm/EVMCodeTransform.cpp

```diff
@@ -728,7 +723,8 @@ size_t CodeTransform::variableHeightDiff(Scope::Variable const& _var, YulString
 			to_string(heightDiff - limit) +
 			" slot(s) too deep inside the stack."
 		);
-		BOOST_THROW_EXCEPTION(m_stackErrors.back());
+		m_assembly.markAsInvalid();
+		return _forSwap ? 2 : 1;
```

Comment:
What exactly is `_forSwap`? The header doesn't say it either

Reply:
I'll add docs